Libraries/Components/AppleTV/TVEventHandler.js

```diff
@@ -1,51 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * @format
- * @flow
- */
-
-'use strict';
-
-import NativeEventEmitter from '../../EventEmitter/NativeEventEmitter';
-import Platform from '../../Utilities/Platform';
-import {type EventSubscription} from '../../vendor/emitter/EventEmitter';
-import NativeTVNavigationEventEmitter from './NativeTVNavigationEventEmitter';
-
-class TVEventHandler {
```

Comment:
@EyMaddis Are you sure? `TVEventHandler` component was located in the `AppleTV` directory and it is based on the `TVNavigationEventEmitter` which seems to be iOS only.

Reply:
Yes, I am sure. I am building an Android TV App right now and the TVEventHandler gets the key events and focus/blur.

Android sends the events here:
https://github.com/facebook/react-native/blob/aee88b6843cea63d6aa0b5879ad6ef9da4701846/ReactAndroid/src/main/java/com/facebook/react/ReactAndroidHWInputDeviceHelper.java#L91

NativeTVNavigationEventEmitter.js uses the native module `TVNavigationEventEmitter` which is defined here: 
https://github.com/facebook/react-native/blob/4a1820dbdf999548772701da9f33d2b9a9304389/ReactAndroid/src/main/java/com/facebook/fbreact/specs/NativeTVNavigationEventEmitterSpec.java

I cannot seem to find the place where the NativeTVNavigationEventEmitterSpec gets integrated, but I can certainly say that TVEventHandler works just fine on Android TV and is crucial to its functionality.

Reply:
@EyMaddis Thank you for your feedback! As I have wrote in this thread before, I don't have much experience with building for Android TVs and I'm glad that you have shared your concerns. I will try to investigate this.

Can you recommend me any open source Android TV app build with RN, so I can test the changes from this branch locally? 

BTW. It would be nice to have an Android TV Pressable related tests in the RNTester (I assume those tests are missing, since CI run was successful).

Reply:
Hmm... I tried google-fu myself, but could only find very old projects.
I think you should be able to just run an android tv emulator with the RNTester app, all Touchables should be navigatable with dpad automatically.

Reply:
@Simek @EyMaddis I'm sorry I'm so late to this conversation -- I'm just now getting to the work of moving the React Native TV repo to 0.64.2 and saw this PR and discussion.  Yes `TVEventHandler` is a weird API, but it's much easier if you use my more modern version, `useTVEventHandler()` (see https://github.com/react-native-tvos/react-native-tvos/pull/86).  Maybe that could be merged to core.